.formatter.exs

```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
+[
+  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
+]
```

Comment:
Was all the code formatted with the formatter? I'd be fine with a change like this

Reply:
> Was all the code formatted with the formatter? I'd be fine with a change like this

@michalmuskala Nope. You want it within the same PR or separate PR?